tmt/steps/report/reportportal.py

```diff
@@ -91,77 +139,222 @@ class ReportReportPortal(tmt.steps.report.ReportPlugin[ReportReportPortalData]):
 
     def handle_response(self, response: requests.Response) -> None:
         """
-        Check the server response and raise an exception if needed.
+        Check the endpoint response and raise an exception if needed.
         """
 
         if not response.ok:
             raise tmt.utils.ReportError(
                 f"Received non-ok status code from ReportPortal: {response.text}")
 
-        self.debug("Response code from the server", response.status_code)
-        self.debug("Message from the server", response.text)
+        self.debug("Response code from the endpoint", str(response.status_code))
+        self.debug("Message from the endpoint", str(response.text))
```

Comment:
Neither of these `str()` calls should be needed, recent changes in `debug()` now support `int` values out of the box (`status_code`), and `text` is already a string.